tests/expressions/tests.py

```diff
@@ -1367,18 +1367,28 @@ def test_functions(self):
 
     def test_aggregates(self):
         self.assertEqual(repr(Avg('a')), "Avg(F(a))")
-        self.assertEqual(repr(Count('a')), "Count(F(a), distinct=False)")
-        self.assertEqual(repr(Count('*')), "Count('*', distinct=False)")
+        self.assertEqual(repr(Count('a')), "Count(F(a))")
+        self.assertEqual(repr(Count('*')), "Count('*')")
         self.assertEqual(repr(Max('a')), "Max(F(a))")
         self.assertEqual(repr(Min('a')), "Min(F(a))")
         self.assertEqual(repr(StdDev('a')), "StdDev(F(a), sample=False)")
         self.assertEqual(repr(Sum('a')), "Sum(F(a))")
         self.assertEqual(repr(Variance('a', sample=True)), "Variance(F(a), sample=True)")
 
+    def test_distinct_aggregates(self):
+        self.assertEqual(repr(Avg('a', distinct=True)), "Avg(F(a), distinct=True)")
+        self.assertEqual(repr(Count('a', distinct=True)), "Count(F(a), distinct=True)")
+        self.assertEqual(repr(Count('*', distinct=True)), "Count('*', distinct=True)")
+        self.assertEqual(repr(Max('a', distinct=True)), "Max(F(a), distinct=True)")
+        self.assertEqual(repr(Min('a', distinct=True)), "Min(F(a), distinct=True)")
+        self.assertEqual(repr(StdDev('a', distinct=True)), "StdDev(F(a), distinct=True, sample=False)")
+        self.assertEqual(repr(Sum('a', distinct=True)), "Sum(F(a), distinct=True)")
+        self.assertEqual(repr(Variance('a', distinct=True, sample=True)), "Variance(F(a), distinct=True, sample=True)")
```

Comment:
Should we add tests for `distrinct=True` and `filter=Q(...)`? Not sure if it's needed, just wondering.

Reply:
Yeah definitely! Will do!

Reply:
Handled below.